portal/templates/redesign/teach_new/onboarding_school.html

```diff
@@ -15,6 +15,13 @@
 {% block content %}
 {{ block.super }}
 
+<script type="text/javascript" src="{% static 'portal/js/fuzzySchoolLookup.js' %}"></script>
+<script>
+    var FUZZY_NAME_FIELD_ID = '{{ join_form.fuzzy_name.auto_id }}';
```

Comment:
I know this is really the thing in the CFL codebase, but it is better not to leave these things lying around in the global scope.  At least create an object to put them in.